autotst/conformer/systematic.py

```diff
@@ -213,25 +217,42 @@ def opt_conf(conformer, calculator, i):
 
             calculator.atoms = conformer.ase_molecule
 
-        from ase.constraints import FixBondLengths
-        c = FixBondLengths(labels)
-        conformer.ase_molecule.set_constraint(c)
-
         conformer.ase_molecule.set_calculator(calculator)
-
         opt = BFGS(conformer.ase_molecule, logfile=None)
-        opt.run(fmax=0.1)
+
+        if type == 'species':
```

Comment:
I think fixing the bond lengths is important for TS since we are trying to optimize a saddle point, and the reactants and products are downhill.  I don't think its needed for molecules though because we are optimizing to a local minima.  The bond lengths based on the forcefield embedding may not be the optimal length at the DFTB level, and when I took the bond constraints off, the optimizations were much faster.

Reply:
We could try to optimize without constraints for molecules.  If the molecule is not isomorphic after opt, we could then apply the constraints and optimize to ensure that its isomorphic

Reply:
Well, in this step, we're walking both the species *_and_* the TS downhill. The only difference is that we keep the reaction center fixed in addition to all actual bonds. So, both of these are going to minima. I'm still leaning towards keeping the constraints to err on the side of caution. This process is pretty cheap with DFTB or force field calculators compared to later steps. 

@rwest do you have any input?

Reply:
But I do think the isomorphism check would be useful even if we do or do not get rid of the bond constraints

Reply:
We sounds definitely be checking for isomorphism after optimization. I like the idea of, if isomorphism check fails, trying again with the bonds constrained. 
I think letting them relax a bit should usually help get to a better starting geometry for the next step. It's only when it "reacts" during optimization that's problematic